src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -1546,6 +1546,13 @@ type Itinerary {
     """
     accessibilityScore: Float
 
+    """
+    How many transfers are contained in this itinerary.
+
+    Note: Interlined/stay-seated transfers also increase the count.
+    """
+    numberOfTransfers: Int
```

Comment:
I wonder if this could be made non null? Is there a potential case where we can't tell how many transfers there are or does it not make sense to tell there are 0 transfers for a direct itinerary?

Reply:
We can always tell so this can be non-nullable.